src/main/java/com/spring/studentmanagement/controllers/UserController.java

```diff
@@ -52,5 +55,19 @@ public String deleteUser(@RequestParam("userId") Long userId) {
     }
 
 
+    @PostMapping(path = "/register")
+    public ResponseEntity<?> registerUser(@RequestBody AppUser user) {
```

Comment:
We don't need to return a ResponseEntity. We return a View.
ResponseEntity are used in the RestControllers